pycqed/measurement/detector_functions.py

```diff
@@ -2026,3 +2296,20 @@ def get_values(self):
     def finish(self):
         if self.AWG is not None:
             self.AWG.stop()
+
+
+class RTO1024_detector(Hard_Detector):
```

Comment:
This class should not go in this pull request, as it uses the scope driver that is under development, has not been tested yet, and has nothing to do with the UHFQC detectors.